CONTRIBUTING.md

```diff
@@ -0,0 +1,75 @@
+- [Pull Requests](https://github.com/InstituteforDiseaseModeling/covasim/pulls) for on going code and documentation contributions. 
+- Readme file contain relevant information for the included components.
+	- [Project Readme](https://github.com/InstituteforDiseaseModeling/covasim/blob/master/README.md) - Setup and project overview.
+	- [Covasim Readme](https://github.com/InstituteforDiseaseModeling/covasim/blob/master/covasim/README.md) Simulation parameters.
+	- [Web App Readme](https://github.com/InstituteforDiseaseModeling/covasim/blob/master/covasim/webapp/README.md) Running the web application.
+	- [Docker Readme](https://github.com/InstituteforDiseaseModeling/covasim/blob/master/docker/README.md) Information regarding the docker setup.
+	- [Tests Readme](https://github.com/InstituteforDiseaseModeling/covasim/tree/master/tests) Running tests locally.
+- [Changelog](https://github.com/InstituteforDiseaseModeling/covasim/blob/master/CHANGELOG.md) 
+
+## Contribution Types
+This is a fast moving project with many opportunities to contribute across the project. We welcome the following types of contributions:
+
+1. Issues:
+	1. Bug reports. 
+	1. Feature requests.
+1. Pull Requests:
+	1. Tests reproducing an issue. 
+	1. Bug fixes.
+	1. Code to resolve [open approved issues](https://github.com/InstituteforDiseaseModeling/covasim/issues?q=is%3Aopen+is%3Aissue+label%3Aapproved).   
+	1. Documentation improvements.
+
+All external communication about these contribution efforts is currently occurring on GitHub.
+
+## Opening Issues
+Before opening a new issue:
+
+* **check your parameters [the README](https://github.com/InstituteforDiseaseModeling/covasim/blob/master/covasim/README.md)** to see if the behavior you observed might be expected and if configuration options are available to provide you with the desired behavior.
+* **perform a cursory search** to see if there's [an existing issue](https://github.com/InstituteforDiseaseModeling/covasim/issues) covering your feedback. If there is one and the issue is still open, **add a :+1: reaction** on the issue to express interest in the issue being resolved. That will help the team gauge interest without the noise of comments which trigger notifications to all watchers. Comments should be used only if you have new and useful information to share.
+
+When opening an issue for a feature request:
+
+* **use a clear and descriptive title** for the issue to identify the problem.
+* **include as many details as possible in the body**. Explain your use-case, the problems you're hitting and the solutions you'd like to see to address those problems.
+
+When opening an issue for a bug report, explain the problem and include additional details to help maintainers reproduce the problem:
+
+* **describe the exact steps which reproduce the problem** in as many details as possible. When listing steps, don't just say what you did, but explain how you did it.
```

Comment:
Issue template could have this all, which would dramatically shorten this document

Reply:
even with issue templates, i don't think it hurts to have duplication -- i'd leave as is

Reply:
Yes, I mentioned that in chat and the related issue.